tests/nix_tests/test_cli.py

```diff
@@ -63,3 +63,75 @@ def test_block_cmd(evmos_cluster):
     )
     # check if chain continues alright
     wait_for_block(node1, last_block + 3)
+
+
+# TODO: why is the signer1 account not found?
```

Comment:
@GAtom22 not sure why the signer1 is not found - it's included in the `default.jsonnet` file and from what I can tell, that's what the network genesis is built upon?